src/layouts/components/Homepage/AnnouncementBody.tsx

```diff
@@ -78,7 +78,22 @@ export const AnnouncementBody = ({
             announcements are shown on the top of the list`}
           </Text>
 
-          <Editable.Accordion onChange={() => onDisplay("announcement")}>
+          <Editable.Accordion
+            defaultIndex={0}
+            onChange={(idx: ExpandedIndex) => {
+              if (typeof idx === "number") {
+                onDisplay("announcement", idx)
+              } else if (
+                idx instanceof Array &&
+                idx.length > 0 &&
+                typeof idx[0] === "number"
+              ) {
+                onDisplay("announcement", idx[0])
+              } else {
+                onDisplay("announcement", -1)
+              }
+            }}
+          >
```

Comment:
could we just add a comment here? alternatively, something we could do is change the API of `Accordion` (since we **do not** allow multiple) but it seems out of scope of this PR

Reply:
ya you are right this is abit confusing 8a7a0af058296742d3c7e252b404d4835afabadb